aiosmtpd/smtp.py

```diff
@@ -35,6 +42,8 @@ def __init__(self, handler,
                  enable_SMTPUTF8=False,
                  decode_data=False,
                  hostname=None,
+                 tls_context=None,
+                 require_starttls=False,
                  loop=None):
```

Comment:
I wonder if we should make `loop` a required keyword argument?  We keep inserting new parameters just before it, and that technically breaks the API.  Of course, we're still alpha so it's not that big of a deal, but what do you think of doing something like `require_starttls=False, *, loop=None` (suitably formatted)?

Then as part of the permanent API, if you specify `loop` you must do so as a keyword argument, and adding more positional arguments before it won't break the API.